src/layouts/DialogLayout/DialogLayout.stories.tsx

```diff
@@ -0,0 +1,131 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Dialog } from '../../components/containers/Dialog/Dialog.tsx';
+import { Logo } from '../../layouts/AppLayout/Logo/Logo.tsx';
+import { H3 } from '../../typography/Heading/Heading.tsx';
+import { Form } from '../../components/forms/context/Form/Form.tsx';
+import { FormLayout } from '../FormLayout/FormLayout.tsx';
+import { CheckboxGroup } from '../../components/forms/controls/CheckboxGroup/CheckboxGroup.tsx';
+import { InputField } from '../../components/forms/fields/InputField/InputField.tsx';
+import { Button } from '../../components/actions/Button/Button.tsx';
+import { Stepper } from '../../components/navigations/Stepper/Stepper.tsx';
+import { Icon } from '../../components/graphics/Icon/Icon.tsx';
+import { FieldLayout } from '../../components/forms/fields/FieldLayout/FieldLayout.tsx';
+
+import { DialogLayout } from './DialogLayout';
+
+type DialogLayoutArgs = React.ComponentProps<typeof DialogLayout>;
+type Story = StoryObj<DialogLayoutArgs>;
+
+export default {
+  tags: ['autodocs'],
+  component: DialogLayout,
+  parameters: {
+    layout: 'padded',
+  },
+} satisfies Meta<DialogLayoutArgs>;
+
+
+export const DialogPattern2: Story = {
+  render: () => {
+    return (
+      <Dialog
+        // not exactly the same logo on Figma, but I guess that's irrelevant for this story.
+        title={<Logo subtitle="Armor"/>}
+        showCancelAction={false}
+      >
+        <DialogLayout
+          title="Let's Connect to Your Cloud Provider"
+          aside={(
+            <>
+              <Stepper
+                steps={[
+                  { stepKey: '1', title: 'Select Cloud Provider' },
+                  { stepKey: '2', title: 'Setup Cloud Connection' },
+                  { stepKey: '3', title: 'Set Up Subscriptions' },
+                  { stepKey: '4', title: '4th Option Title' },
+                ]}
+                activeKey="1"
+                onSwitch={() => {}}
+              />
+              {/* TODO: Maybe this should be a separate component? */}
```

Comment:
Probably not a separate component. Could be a mixin. Or you could use `bk-prose` which has styling for `hr`